widgets/hits.js

```diff
@@ -2,7 +2,12 @@ var React = require('react');
 
 var utils = require('../lib/utils.js');
 
-function hits({container = null, templates = {}, hitsPerPage = 20}) {
+function hits({
+    container = null,
+    templates = {},
+    transformData = {},
```

Comment:
I'll push with `transformData`, and we can reopen this dicussion if this really bothers us, ok with you @bobylito ?